.devcontainer/README.md

```
@@ -0,0 +1,85 @@
+- **Build Production** / **Build Production with React**
+- **Run Tests** / **Run Tests with Coverage**
+- **Lint Code** / **Lint and Fix**
+
+### Debugging
+- **Jest Tests**: Use the "Debug Jest Tests" launch configuration
+- **Current Test File**: Use "Debug Current Jest Test" when a test file is open
+- **Build Scripts**: Use "Debug Build Script" for debugging the build process
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)